charts/kube-state-metrics/Chart.yaml

```
@@ -7,7 +7,7 @@ keywords:
 - prometheus
 - kubernetes
 type: application
-version: 4.4.3
+version: 4.4.4
```

new feature should raise minor version

```suggestion
version: 4.5.0
```

---

Fixed by @this-chris